tests/integration/staking/keeper/msg_server_test.go

```diff
@@ -81,6 +81,17 @@ func TestCancelUnbondingDelegation(t *testing.T) {
 		req       types.MsgCancelUnbondingDelegation
 		expErrMsg string
 	}{
+		{
+			Name:      "entry not found at height",
+			ExceptErr: true,
```

Comment:
I know you didn't do that, but it looks very weird that Name and ExceptErr are exported but the rest not.

Reply:
oh, I didn't really notice it. Good catch julien, I'll address it.